src/sql-parser/src/parser.rs

```diff
@@ -2366,7 +2388,7 @@ impl<'a> Parser<'a> {
                 KafkaConfigOptionName::EnableIdempotence
             }
             FETCH => {
-                self.expect_keywords(&[MESSAGE, crate::keywords::MAX, BYTES])?;
+                self.expect_keywords(&[MESSAGE, mz_sql_lexer::keywords::MAX, BYTES])?;
```

Comment:
Looks like someone wasn't paying attention when they added this code originally! All keywords should be imported automatically.

```suggestion
                self.expect_keywords(&[MESSAGE, MAX, BYTES])?;
```